extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php

```diff
@@ -190,6 +191,36 @@ function () {
 		return $botIds;
 	}
 
+
+	/**
+	 * @return array list of blocked ids for Top Contributors
+	 */
+	private function getBlacklistedIds(){
+		$blacklistedIds = WikiaDataAccess::cache(
+			self::getMemcKey( self::ALL_BLOCKED_IDS_MCACHE_KEY ),
+			WikiaResponse::CACHE_LONG,
+			function () {
+				global $wgExternalSharedDB;
+				$globalDb = wfGetDB( DB_SLAVE, [], $wgExternalSharedDB );
+
+				$globalIds = ( new WikiaSQL() )
+					->SELECT( 'ug_user' )
+					->FROM ( 'user_groups' )
+					->WHERE( 'ug_group' )->IN( [ 'bot', 'bot-global', 'staff', 'util', 'helper', 'vstf' ] )
+					->GROUP_BY( 'ug_user' )
+					->runLoop( $globalDb, function ( &$globalIds, $row ) {
+						$globalIds[] = $row->ug_user;
+					} );
+
+				$allBlockedIds = array_merge( $globalIds, $this->getBotIds() );
+
+				return $allBlockedIds;
```

Comment:
This is still using the "blocked" in the variable name, but I'd suggest just changing it to:
```php
return array_merge( $globalIds, $this->getBotIds() );
```